src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/FixAllContext.DiagnosticProvider.cs

```diff
@@ -179,7 +182,10 @@ private static Document GetReportedDocument(Diagnostic diagnostic, ImmutableDict
             internal virtual async Task<ImmutableDictionary<Project, ImmutableArray<Diagnostic>>> GetProjectDiagnosticsToFixAsync(
                 FixAllContext fixAllContext)
             {
-                using (Logger.LogBlock(FunctionId.CodeFixes_FixAllOccurrencesComputation_Diagnostics, fixAllContext.CancellationToken))
+                using (Logger.LogBlock(
+                    FunctionId.CodeFixes_FixAllOccurrencesComputation_Project_Diagnostics,
+                    KeyValueLogMessage.Create(LogType.UserAction, m => m[FixAllLogger.CorrelationId] = fixAllContext.State.CorrelationId),
```

Comment:
Should this be refactored somewhere vs. repeated everywhere? I don't know, just asking.

Reply:
sure. I can do that.